### DISCUSSION TITLE
Fred had a red rubber

### DISCUSSION THREAD

**Discussion by teeratan** [gautamkrishnar/blog-post-workflow#69] [General]
Fred had a red rubber ball. He kicked the ball. It flew through the air. He picked it up. He threw it against a wall. The ball came back to Fred. He kicked it with his foot

 อ่านต่อได้ที่ :  <a href="http://www.anubanbankha.ac.th">อนุบาลบ้านคา</a>

**Comment by riginoommen** [comment]
താൻ എന്ത് തേങ്ങ ആടോ പോസ്റ്റ് ചെയ്തത്. 

സാർ ഒന്ന് പറഞ്ഞു തരുമൊ
